### PR TITLE
Fixed Critical and High vulnerabilities

### PR DESCRIPTION
**Pull request by dpattermann-usgs**
Should be basically exactly the same as Zack's

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -128,6 +130,21 @@
...
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-core</artifactId>
+            <version>9.0.30</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.tomcat.embed</groupId>
+            <artifactId>tomcat-embed-websocket</artifactId>
+            <version>9.0.30</version>
+        </dependency>
```
Comment (zmoore-usgs) [inline pom.xml]: I think you want these 3 guys up in `dependencyManagement` rather than the top-level `dependencies`

File: pom.xml [hunk 2]
```diff
@@ -166,17 +183,30 @@
...
                         <exclude>gov/usgs/aqcu/Application.class</exclude>
                     </excludes>
-				</configuration>
-			</plugin>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
```
Comment (zmoore-usgs) [inline pom.xml]: I think you need the little version tag here: https://github.com/USGS-CIDA/aqcu-srs-report/pull/20/files#diff-600376dffeb79835ede4a0b285078036R198 to fix non-CI builds from getting a warning about no version being defined

Comment (dpattermann-usgs) [reply pom.xml]: Ach! I grabbed from the wrong PR

File: pom.xml (hunk 1 above)
Comment (dpattermann-usgs) [reply pom.xml]: 🤦‍♂

**Comment by zmoore-usgs** [comment]
Looks good! Feel free to merge!

**Pull request merged**